src/components/mini-widgets/BatteryIndicator.vue

```diff
@@ -99,12 +111,13 @@ const currentDisplayValue = computed(() => {
     : store.powerSupply.current.toFixed(1)
 })
 
-const instantaneousWattsDisplayValue = computed(() => {
-  return store.instantaneousWatts !== undefined ? store.instantaneousWatts.toFixed(1) : '--'
```

Comment:
No need to move those lines (from 102-103 to 119-120). It's creating a false delete->add in git which makes it hard to track changes in the code.